config/precompute.config

```diff
@@ -1,10 +1,13 @@
 params {
   precompute {
     run = true
-    max_entries = 500000
+    max_entries = 250000
     load_size = 1024 * 1000 * 1000
     maxForks = 5
-    method = 'release'
+    method = 'query'
```

Comment:
This change should probably be reverted

Reply:
fixed in https://github.com/RNAcentral/rnacentral-import-pipeline/pull/151/commits/0b434827c3bfb346269735a33d108a8927bdbba2